requirements.txt

```diff
@@ -49,3 +49,4 @@ https://control.divio.com/api/v1/apps/serve/aldryn-django/2.2.16.1/b460938a-f357
 django-import-export
 us==2.0.2
 django-admin-rangefilter==0.6.3
+django-flags==5.0.1 # for feature flags
```

Comment:
@sebgrebe, oh btw the flow for the requirements changes:
- add the requirement to requirements.in
- run `docker-compose run --rm web bash -c 'pip-compile requirements.in > requirements.txt'`

I thought about adding it to the readme but then realized it would be better to move to poetry and store everything in pyproject.toml in any case

Reply:
@victor-yunenko, where is requirements.in? I can't see it among the files in the project

Reply:
Ah right, it's another branch. I would recommend merging it in since I wouldn't want to deploy or test it without frozen requirements.